jobbergate-api/jobbergate_api/apps/job_script_templates/routers.py

```diff
@@ -154,9 +154,11 @@ async def job_script_template_update(
 ):
     """Update a job script template by id or identifier."""
     logger.info(f"Updating job script template {id_or_identifier=} with {update_request=}")
-    await secure_services.crud.template.get(
-        id_or_identifier, ensure_attributes={"owner_email": secure_services.identity_payload.email}
-    )
+    instance = await secure_services.crud.template.get(id_or_identifier)
+    if Permissions.JOB_TEMPLATES_ADMIN not in secure_services.identity_payload.permissions:
```

Comment:
I think we actually want just one admin permission that covers all three entities. If someone is an admin of jobbergate, they should have the same rights on all three.

Reply:
solved in [2576767](https://github.com/omnivector-solutions/jobbergate/pull/551/commits/2576767153a5df1a6e440b435aed62666a144b42)